src/View/TemplatePaths.php

```diff
@@ -666,7 +655,7 @@ protected function getPartialPathAndFilename($partialName) {
 	 */
 	protected function resolveFileInPaths(array $paths, $relativePathAndFilename, $format = self::DEFAULT_FORMAT) {
 		$tried = array();
-		foreach ($paths as $path) {
+		while ($path = array_pop($paths)) {
```

Comment:
You're right - I'll put this as a code comment for anyone reading there and not reading this pull request.